src/components/my-page/IdeaMarketPostHeader.tsx

```diff
@@ -1,19 +1,30 @@
+import { useNavigate } from 'react-router-dom';
 import styles from './ideaMarketPostHeader.module.scss';
-import arrowButton from '../../assets/icons/arrow-button.svg';
 
 interface PostHeaderProps {
   category: string;
   title: string;
   price: number;
+  postId: number;
 }
 
 export const IdeaMarketPostHeader = ({
   category,
   title,
   price,
+  postId,
 }: PostHeaderProps) => {
+  const navigate = useNavigate();
   const FORMATTEDPRICE = price.toLocaleString();
```

Comment:
스네이크 방식으로 사용하시면 가독성면에서 좋아요!

```suggestion
  const FORMATTED_PRICE = price.toLocaleString();
```